tests/system_tests_one_router.py

```diff
@@ -1514,17 +1547,20 @@ def __init__(self, addr, n_messages, begin_ingress_count, begin_dropped_presettl
         self.custom_timer = None
         self.timer = None
         self.begin_dropped_presettled_count = begin_dropped_presettled_count
-        self.begin_ingress_count = begin_ingress_count
         self.str1 = "0123456789abcdef"
         self.msg_str = ""
         self.max_tries = 10
         self.reactor = None
         for i in range(8192):
             self.msg_str += self.str1
         self.timer_instance = PresettledCustomTimeout(self)
+        self.send_timer = SendPresettledAfterReceiverCloses(self)
```

Comment:
two things:  I think self.timer_instance is no longer used, and doesn't self.send_timer need to be None here?